models/repo_avatar.go

```diff
@@ -110,6 +110,9 @@ func (repo *Repository) avatarLink(e db.Engine) string {
 	link := repo.relAvatarLink(e)
 	// link may be empty!
 	if len(link) > 0 {
+		if len(link) == 1 {
```

Comment:
Maybe we should document with a comment when this case can happen and why we return the default URL in that case…

Reply:
Wait, now that I think about it:
The current implementation is not necessarily the correct solution.
If I see that correctly,
```suggestion
		if link == "/" {
```
is potentially better than `len(link) == 1`:
Can we guarantee that the fallback image link might not be i.e. a file named `a` inside the repo?

Reply:
> Can we guarantee that the fallback image link might not be i.e. a file named a inside the repo?

Given common sense? Yes.
Given assuming bad input? No. A user could simply specify something like `.` or `?` or `b` as the setting and still will panic.